resilience4j-framework-common/src/main/java/io/github/resilience4j/common/ratelimiter/configuration/RateLimiterConfigurationProperties.java

```diff
@@ -106,8 +114,8 @@ public InstanceProperties getInstanceProperties(String instance) {
 	public static class InstanceProperties {
 
 		private Integer limitForPeriod;
-		private Integer limitRefreshPeriodInNanos;
-		private Integer timeoutInMillis;
+		private Duration limitRefreshPeriod;
+		private Duration timeout;
```

Comment:
Please reanme to `timeoutDuration` which matches `RateLimiterConfig.timeoutDuration`